presto-main/src/main/java/io/prestosql/sql/DynamicFilters.java

```diff
@@ -129,15 +133,22 @@ public ExtractResult(List<Expression> staticConjuncts, List<Descriptor> dynamicC
 
     public static final class Descriptor
     {
+        private final String sourceId;
         private final String id;
         private final Expression input;
 
-        public Descriptor(String id, Expression input)
+        public Descriptor(String sourceId, String id, Expression input)
         {
+            this.sourceId = requireNonNull(sourceId, "sourceId is null");
             this.id = requireNonNull(id, "id is null");
             this.input = requireNonNull(input, "input is null");
         }
 
+        public String getSourceId()
```

Comment:
this seems unused.

Reply:
It was being used in DynamicFilterSupplier earlier, I've removed it now